src/lib/Scenes/Show/Components/Location/index.tsx

```diff
@@ -1,14 +1,74 @@
+import { color, Flex, Sans } from "@artsy/palette"
+import Mapbox from "@mapbox/react-native-mapbox-gl"
+import { Location_show } from "__generated__/Location_show.graphql"
 import React from "react"
-import { Text } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components/native"
+
+// TODO: This is a testing access token, will need to regenerate and move to cocoapods-keys
+Mapbox.setAccessToken("pk.eyJ1IjoiYXJ0c3lpdCIsImEiOiJjam51dTJibTEwNHNpM3BxamV4dDRibzJhIn0.bhFMJt1mqxNDrE1xSAtxSw")
+
+const Map = styled(Mapbox.MapView)`
+  flex: 1;
+  height: 200;
+`
 
 interface Props {
-  location: any
+  show: Location_show
 }
 
-// TODO: implement
-
 export class Location extends React.Component<Props> {
   render() {
-    return <Text>{JSON.stringify(this.props.location)}</Text>
+    const { location, partner } = this.props.show
+    const { lat, lng } = location.coordinates
+
+    return (
+      <Flex>
+        <Map
+          styleURL={Mapbox.StyleURL.Light}
+          centerCoordinate={[lng, lat]}
+          zoomLevel={13}
+          logoEnabled={false}
+          scrollEnabled={false}
```

Comment:
Not sure if it's explicitly specified in the comps, but for static map behavior would we want to set `zoomEnabled` to false as well?

Reply:
I think it defaults to `false` (even though the docs say `none`) But I'll set it anyway just in case

https://github.com/mapbox/react-native-mapbox-gl/blob/master/docs/MapView.md